capacity4more/modules/c4m/content/c4m_content_organisation/c4m_content_organisation.install

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @file
+ * Installation hooks for the Content Organisation module.
+ */
+
+/**
+ * Implements hook_field_schema().
+ */
+function c4m_content_organisation_field_schema($field) {
+  switch ($field['type']) {
+    case 'c4m_domain':
+      $columns = array(
+        'domain' => array(
+          'type'      => 'varchar',
+          'length'    => 255,
+          'not null'  => TRUE,
+        ),
+        'icon' => array(
+          'type'      => 'int',
+          'length'    => 4,
```

Comment:
Length is only for char, varchar and text fields, size is for int field types.

Remove from definition.
Do not add size, the int will be by default the correct value.